v4/export/writer_util.go

```diff
@@ -114,6 +120,64 @@ func (b *writerPipe) ShouldSwitchStatement() bool {
 		(b.statementSizeLimit != UnspecifiedSize && b.currentStatementSize >= b.statementSizeLimit)
 }
 
+// SpeedLimiter used for control speed of dump data
+type SpeedLimiter struct {
+	count  int
```

Comment:
What's the meaning of `count`? Why do we sleep less when we are about to `flush` `limiter.size`?

Reply:
I adjusted the whole thing.